src/renderer/shared/ui/Identicon/Identicon.tsx

```diff
@@ -17,14 +17,16 @@ type Props = {
 };
 
 export const Identicon = ({
-  theme = 'polkadot',
+  theme,
   address,
   size = 24,
   background = true,
   canCopy = true,
   className,
   buttonClassName,
 }: Props) => {
+  const valueTheme = theme || (address && isEthereumAccountId(address as AccountId) ? 'ethereum' : 'polkadot');
```

Comment:
Ideally we should pass already defined `theme` instead of calculating it inside UI component
because it should know nothing about the rules we apply to get `eth` or `dot` theme 🤔 (add a comment please)
But for now it's ok

Reply:
It's something to identify default theme by address, because we don't use theme in some places, rewrite it a little to better explain this idea